lib/content/audio/train_is_boarding.ex

```diff
@@ -53,7 +53,8 @@ defmodule Content.Audio.TrainIsBoarding do
                 :government_center,
                 :park_st,
                 :kenmore,
-                :union_square
+                :union_square,
+                :medford_tufts
```

Comment:
I believe there are very few, if any, cases where we announce boarding trains for Green line so this change may not be necessary. But I decided to include it for good measure.